ha/core/event_analyzer/parser/parser.py

```diff
@@ -168,6 +169,15 @@ def __init__(self):
         self.cluster_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
         self.site_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}site_id")
         self.rack_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}rack_id")
+
+        #from ha.core.config.config_manager import ConfigManager
+        #ConfigManager._safe_load("cortx", "yaml:///etc/cortx/cluster.conf")
+        #ConfigManager.init(None)
+        _index = 'cortx'
+
```

Comment:
I don't think cluster.conf is loaded and available during service run time.
Better check ha conf_store.

Reply:
correct, 
In ConfStoreSearch class, cluster.conf is not loaded.
Donot remove it from here.

Also,
#L174, 

> #ConfigManager._safe_load("cortx", "yaml:///etc/cortx/cluster.conf")

remove 'cortx' with a variable '_index'

Reply:
yes, @mariyappanp  I checked with ha_setup command also but still required load of cluster.conf file